bump-version/index.js

```diff
@@ -15,14 +15,26 @@ class BumpVersion extends Action_1.Action {
     async onTriggered(octokit) {
         const { owner, repo } = github_1.context.repo;
         const token = this.getToken();
-        const payload = github_1.context.payload;
-        const version = payload.inputs.version;
-        if (!version) {
-            throw new Error('Missing version input');
-        }
         await (0, git_1.cloneRepo)({ token, owner, repo });
         process.chdir(repo);
-        const base = github_1.context.ref.substring(github_1.context.ref.lastIndexOf('/') + 1);
+        if (!this.isCalledFromWorkflow()) {
+            // Manually invoked the action
+            const version = this.getVersion();
+            const base = github_1.context.ref.substring(github_1.context.ref.lastIndexOf('/') + 1);
+            await this.onTriggeredBase(octokit, base, version);
+            return;
+        }
+        // Action invoked by a workflow
+        const version_call = this.getVersion();
+        const matches = version_call.match(/^(\d+.\d+).\d+(?:-beta.\d+)?$/);
```

Comment:
🤔 I think here the regex for beta needs to change, if we are passing the milestone version from the reusable workflow, it means we will pass something like "[7.0.0-beta1](https://github.com/grafana/grafana/milestone/156)", it might be worthy to change the error message as well.

```suggestion
        const matches = version_call.match(/^(\d+.\d+).\d+(?:-beta\d+)?$/);
```

Reply:
Hmm, no for bump version it really needs to be a semantic version like x.x.x-beta.1, x.x.x-beta.2. But I think you find something else here. The remove/close-milestone actions might need a slight change. Maybe even the update changelog. 🤔

Reply:
On second thought might be easier to just change the bump version one as you suggested 🙈

Reply:
Yeah, I was thinking this change will not be conflicting, because we only need to get the first part of the milestone to build the base branch:

milestone (version_call): `8.0.0-beta1`
our match will be:` 8.0`
base branch: `8.0.x`

Reply:
Indeed, but when we bump the version in `onTriggerBase` the `version_call` param above needs to be converted to `8.0.0-beta.1`. Then it should work as expected.

Reply:
oh yes! I was missing that part